@here/harp-mapview/lib/DataSource.ts

```diff
@@ -440,10 +441,9 @@ export abstract class DataSource extends THREE.EventDispatcher {
      *
      * @deprecated Use [[setTheme]].
      *
-     * @param styleSet - The new theme that {@link MapView} uses.
-     * @param languages - An optional list of languages for the `DataSource`.
+     * @param options - The Options used for decodeing and/or styling
      */
-    setStyleSet(styleSet?: StyleSet, definitions?: Definitions, languages?: string[]): void {
```

Comment:
I am not so convinced of this new signature,... but as its deprecated and heading into using setTheme instead, which then would contain all the parameters anyway ( I would do that in a next PR, as this one is so big already) , I think it could be acceptable for now??